src/org/usfirst/frc/team166/robot/commands/GearManipulator/ManipulatorMotorOff.java

```diff
@@ -0,0 +1,43 @@
+package org.usfirst.frc.team166.robot.commands.GearManipulator;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+import org.usfirst.frc.team166.robot.Robot;
+
+/**
+ *
+ */
+public class ManipulatorMotorOff extends Command {
```

Comment:
For future reference: there's an `InstantCommand that gets rid of some of this boilerplate